packages/api-client/src/helpers/findVariables.ts

```diff
@@ -6,7 +6,7 @@ export const findVariables = (value: string) => {
   // Wrapped in single or double curly braces.
   // Ignores whitespace
   // Works with lowercase, uppercase, numbers, dashes, underscores
-  const regex = /(?<=\{)\s*\w+\s*(?=\})/g
+  const regex = /(?=\{)\s*\w+\s*(?=\})/g
```

Comment:
So safari doesn't support look behind until version 16.4 and I was running 16.3, do we need this?

https://caniuse.com/js-regexp-lookbehind